### ISSUE TITLE
mark-callable-contracts rule is firing a false positive on enums 

### ISSUE THREAD

**Issue by adetante** [protofire/solhint#133]
The mark-callable-contracts rule is being triggered against enums (similar to #114 and #118).

Simple contract to reproduce the issue:
```
pragma solidity 0.5.9;

contract Example {

    enum Status { Initial }

    Status public status;

    constructor() public {
        status = Status.Initial;
    }
  
}
```

Produces the following warnings:
```
10:18  warning  Explicitly mark all external contracts as trusted or untrusted  mark-callable-contracts
10:25  warning  Explicitly mark all external contracts as trusted or untrusted  mark-callable-contracts
```

**Comment by adetante** [comment]
I have just created the PR https://github.com/protofire/solhint/pull/134
